### ISSUE TITLE
Lotka volterra UDE not recovered as per paper

### ISSUE THREAD

**Issue by yewalenikhil65** [ChrisRackauckas/universal_differential_equations#39]
`scenario_2.jl` outputs in following fit for UDE.. `scenario_1.jl` also does does not fit (I remember in early versions, this examples were able to recover the model accurately.. what changed ?)

![image](https://user-images.githubusercontent.com/26343789/160559162-89f9f176-1506-489f-9025-54d613329aa6.png)

**Comment by AlCap23** [comment]
On it.

**Comment by AlCap23** [comment]
Could you provide your `]st` for reference? 

Also, there is a specific tag for the paper version, which might work better.

**Comment by AlCap23** [comment]
https://github.com/ChrisRackauckas/universal_differential_equations/releases/tag/v1.0

**Comment by yewalenikhil65** [comment]
> https://github.com/ChrisRackauckas/universal_differential_equations/releases/tag/v1.0

Doesnt look like i can precompile after instantiate this

**Comment by AlCap23** [comment]
Yeah, there is definitely something 🐟 -y going on here. I experienced a full out fail on 1.7 while creating the `ContinuousDataDrivenProblem`, which should not happen. 

Additionally, the network performs really bad. 

I will retry on 1.6 and report a little more over the course of the day.
